index.html

```diff
@@ -11,86 +11,139 @@
 
-OUTSKIRTS
-
-The Outskirts are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. 
-
-Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.
-
-THE BLOCKS
-
-The Blocks are amazing to the table win-win survival strategies to ensure proactive domination. At the end of the day, going forward, a new normal that has evolved from generation X is on the runway heading towards a streamlined cloud solution. 
-
-Capitalize on low hanging fruit to identify a ballpark value added activity to beta test. Override the digital divide with additional clickthroughs from DevOps. Nanotechnology immersion along the information highway will close the loop on focusing solely on the bottom line.
-
--->
-
-<!-- FOOTER CONTENT
-Interested in starting a project?  
-Let’s talk:
-
-Enter email
-
-We'll never share your email with anyone else.
-
-New York
-123 Lane
-Suite 100
-Albany, NY 12345
-202 555 0144
-
-Florida
-Ocean Drive
-Suite 201
-Orlando, FL 22345
-502 555 0144
-
-California
-Mountain Street
-Suite 105
-San Diego, CA 22345
-702 555 0144
-
-Copyright © 2018 Smith and Jones
--->
+	<!-- Home Page Content -->
+	<div class="upper-block">
+
+		<img class="desk" src="img/home/home-jumbotron.png">
+		<img class="mob" src="img/home/home-mobile-jumbotron.png">
+		<p>	
+			Integrity, <br>
+			Excellence, <br>
+			Progress.
+		</p>
+	</div>
+	<div class="main-content"> <!-- Main Content -->
+		<section class="block1">
+			<div class="block1-left">
+				<h2>Smith & Jones Architects</h2>
+				<p>Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam laboramus dissentiet in, nec no errem</p> 
+				<div class=btn>Learn More</div>
+			</div>
+			<div class="block1-right">
+				<img class="desk" src="img/home/home-img-1.png">
+				<img class="mob" src="img/home/home-mobile-img-1.png">
+			</div>
+		</section>
+		<section class="block2">
+			<div class="block2-left">
+				<img class="desk" src="img/home/home-img-2.png">
+				<img class="mob" src="img/home/home-mobile-img-2.png">
+			</div>
+			<div class=block2-right>
+				<h2>Futuristic Designs</h2>
+				<p>Et sed autem causae appareat, tempor abhorreant te mei, facer facilisis sit ea. Eu timeam vidisse consectetuer sed. Duo etiam laboramus dissentiet in, nec no errem</p> 
+				<div class="btn">View Designs</div>
+			</div>
+		</section>
+		<div class="h2Header"><h2>Recent Projects</h2></div>
+		<section class="projects theVillas">
```

Comment:
This probably doesn't need its own class name. You should be able to target it with nesting.